actix-cors/src/inner.rs

```diff
@@ -27,6 +27,12 @@ impl Default for OriginFn {
     }
 }
 
+impl PartialEq for OriginFn {
+    fn eq(&self, other: &Self) -> bool {
+        Rc::ptr_eq(&self.boxed_fn, &other.boxed_fn)
```

Comment:
If this equality here is correct is something that needs debating.
Is function equivalence for you:
- (current) literal => i.e. the pointers need to match
- semantic => i.e. they have to do the same thing

(If it should be semantic, we can close this PR and the linked issue, as I don't think that this is possible)

Reply:
point equality seems fine here